backend/src/app/core/data/crud/code.py

```diff
@@ -159,6 +162,11 @@ def exists_by_name_and_user_and_project(
             is not None
         )
 
+    def update(self, db: Session, *, id: int, update_dto: CodeUpdate) -> CodeORM | None:
+        if update_dto.parent_code_id == -1:
```

Comment:
Is there a way to send `undefined`/`None` from the frontend directly? That might make the code easier to understand since `-1` as a special value is not documented in the openapi spec or elsewhere.